src/SentryFullStory.ts

```diff
@@ -55,25 +55,37 @@ class SentryFullStory {
         }
       };
 
-      const self = Sentry.getCurrentHub().getIntegration(SentryFullStory);
-      // Run the integration ONLY when it was installed on the current Hub AND isn't a transaction
-      if (self && event.type !== 'transaction' && this.doesFullStoryExist()) {
+      const getFullStoryUrl = (): string => {
         // getCurrentSessionURL isn't available until after the FullStory script is fully bootstrapped.
         // If an error occurs before getCurrentSessionURL is ready, make a note in Sentry and move on.
         // More on getCurrentSessionURL here: https://help.fullstory.com/develop-js/getcurrentsessionurl
+        try {
+          return FullStory.getCurrentSessionURL(true) || 'Current session URL API not ready'
+        } catch (e) {
+          if (e instanceof Error) {
+            return `Unable to get url: ${e.message}`
+          }
+          return 'Unable to get url'
+        }
+      }
+
+      const self = Sentry.getCurrentHub().getIntegration(SentryFullStory);
+      // Run the integration ONLY when it was installed on the current Hub AND isn't a transaction
+      if (self && event.type !== 'transaction' && SentryFullStory.doesFullStoryExist()) {
         event.contexts = {
           ...event.contexts,
           fullStory: {
-            fullStoryUrl:
-              FullStory.getCurrentSessionURL(true) ||
-              'current session URL API not ready',
+            fullStoryUrl: getFullStoryUrl()
           },
         };
-        // FS.event is immediately ready even if FullStory isn't fully bootstrapped
-        FullStory.event('Sentry Error', {
-          sentryUrl: getSentryUrl(),
-          ...util.getOriginalExceptionProperties(hint),
-        });
+        try {
+          FullStory.event('Sentry Error', {
+            sentryUrl: getSentryUrl(),
+            ...util.getOriginalExceptionProperties(hint),
+          });
+        } catch (e) {
+          console.debug('Unable to report sentry error details to FullStory')
+        }
```

Comment:
Is there a reason this is `console.debug` vs `console.error`?

Reply:
I tend to only use console.error for things that result in user facing issues & this isn't a particularly informative message.
Happy to consider changes here if you have a suggestion.

Reply:
No suggestion, just curious for the reasoning, looks fine to me.